components/MusicPlayer/MusicPlayer.vue

```diff
@@ -60,6 +77,25 @@
         return `${minutes}:${(seconds % 60).toString().padStart(2, "0")}`;
     }
 
+    function updateProgress() {
+        const state = playbackState.value;
+        if (!(state?.progress_ms && state?.item?.duration_ms)) {
+            return;
+        }
+
+        let progress = state.progress_ms;
+
+        if (state.is_playing) {
+            progress = state.progress_ms + (Date.now() - songProgress.value.fetched);
+        }
+
+        songProgress.value = {
+            progress,
```

Comment:
Kannst du das in einem terniary statement Zusammenfassen? `(condition ? then : else)`

Reply:
ternary statements verwende ich eig. nur wenn bereits fertige variable oder so habe. nicht, wenn noch etwas berechnet werden muss.
`cond ? varA : varB` erlaube ich
`cond ? varA + varB : varA - varB` mag ich nicht

Reply:
Fair enough